### PR TITLE
Fixes #17614 - Update nightly docs from capsule to smart proxy

### PR DESCRIPTION
**Pull request by zjhuntin**
This should only be merged after we branch Katello next, and we have the changes renaming all mentions of Capsule to Proxy or Smart proxy.

### CODE REVIEW

File: plugins/katello/nightly/installation/smart_proxy.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+## Installation
+
+### Install needed packages:
+
+The same yum repositories need to be configured on the Smart proxy server as the main Katello server. See the installation guide for the [list of required repositories](/plugins/katello/{{ page.version }}/installation/index.html#required-repositories).
+
+Once you get the repositories configured, install the katello-capsule package on the smart proxy
+
+{% highlight bash %}
+yum install -y katello-capsule
```
Comment (ehelms) [inline plugins/katello/nightly/installation/smart_proxy.md]: This got changed to foreman-proxy-content

File: plugins/katello/nightly/installation/smart_proxy.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+See the [User Guide](/plugins/katello/{{ page.version }}/user_guide/smart_proxies/index.html) for additional information about Smart proxy services and required ports.
+
+## Installation
+
+### Install needed packages:
+
+The same yum repositories need to be configured on the Smart proxy server as the main Katello server. See the installation guide for the [list of required repositories](/plugins/katello/{{ page.version }}/installation/index.html#required-repositories).
+
+Once you get the repositories configured, install the katello-capsule package on the smart proxy
```
Comment (ehelms) [inline plugins/katello/nightly/installation/smart_proxy.md]: This got changed to foreman-proxy-content

**Review by ehelms — CHANGES_REQUESTED** [review]

**Comment by zjhuntin** [comment]
@ehelms Updated!

**Review by ehelms — APPROVED** [review]

**Comment by ehelms** [comment]
Thanks @zjhuntin !

**Comment by ehelms** [comment]
Actually, @zjhuntin , I spoke too soon, I forgot about this guy: https://github.com/theforeman/theforeman.org/blob/gh-pages/_includes/plugins/katello/sidebar.html

Which you may notice has Capsule in it but is also shared amongst Katello versions currently. I think we will want to split that up so we have a version per Katello version and include that appropriately.

**Comment by zjhuntin** [comment]
@ehelms okay, I'll get working on it

**Comment by zjhuntin** [comment]
@ehelms Okay, I got it to work doing it this way.

File: _layouts/plugins/katello/guide.html [hunk 3]
```diff
@@ -0,0 +1,30 @@
+---
```
Comment (ehelms) [inline _layouts/plugins/katello/guide.html]: Where is this file used at?

File: plugins/katello/nightly/user_guide/smart_proxies/index.md [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+    * Puppet Master servers
+    * Puppet CA to manage certificate signing and cleaning
+    * Baseboard Management Controller (BMC) for power management
+    * Provisioning template proxy
+
+The Katello Smart proxy server is a means to scale out the Katello installation. Organizations can create various smart proxies in different geographical locations. These are centrally managed through the Katello server. When a Katello user promotes content to a particular environment, the Katello server will push the content to each of the Smart proxy servers subscribed to that environment. Hosts pull content and configuration from the Katello Smart proxy servers in their location and not from the central server.
+
+In a fully configured smart proxy, communication is completely isolated between hosts and the Katello server.
+
+## How do Smart proxies differ from Smart Proxies?
```
Comment (ehelms) [inline plugins/katello/nightly/user_guide/smart_proxies/index.md]: This title no longer makes sense -- how about `## What is a Foreman Proxy with Content?`

Comment (zjhuntin) [reply plugins/katello/nightly/user_guide/smart_proxies/index.md]: I had a good laugh at this.

**Review by ehelms — CHANGES_REQUESTED** [review]
I found Capsule here -- https://github.com/theforeman/theforeman.org/pull/769/files#diff-65c3065680a6bc8af4a60ebb811402f3R72

I think it should be 'Smart Proxy' everywhere we talk about it sense we treat it as a proper noun.

**Comment by zjhuntin** [comment]
@ehelms made sure all instances of Smart Proxy are treated as a proper noun.

**Comment by ehelms** [comment]
Looks good - thanks @zjhuntin !

**Review by ehelms — APPROVED** [review]

**Pull request merged**